mmdet3d/core/visualizer/image_vis.py

```diff
@@ -56,46 +56,31 @@ def project_pts_on_img(points,
     cv2.waitKey(100)
 
 
-def project_bbox3d_on_img(bboxes3d,
```

Comment:
Shall we keep the original one? I found `project_bbox3d_on_img` not used and maybe it's fine to keep it here and add the new function instead.

Reply:
I think its name is very ambiguous as we already have `project_camera/lidar/depth_bbox` now. I suggest removing it because its functionality is the same as `draw_lidar_bbox3d_on_img`. You can compare the two functions and have a look :)

Reply:
I agree.